docs/src/pages/system/advanced/advanced.md

```diff
@@ -0,0 +1,21 @@
+# Advanced
+
+<p class="description">Here you can find examples of how you can use the system on your custom components.</p>
+
+## Adding the `sx` prop on your custom components
+
+The `unstable_styleFunctionSx` utility adds the support for the `sx` in your own components.
+
+{{"demo": "pages/system/advanced/StyleFunctionSxDemo.js"}}
+
+### Built-in `sx` from the core
+
+If you are already using the Material-UI's `styled()` wrapper, the easiest way to support the `sx` prop on your custom components is to use the `experimentalStyled()` utility that comes from `@material-ui/core/styles` for creating your custom components. Here is an example of how it can be used:
+
+{{"demo": "pages/system/advanced/ExperimentalStyledDemo.js", "defaultCodeOpen": true}}
+
+## Using standalone system utilities
+
+If you wouldn't like to use the `sx` prop on your custom components, but only some bits from the system, you can use and combine the different style functions available as part of the system.
```

Comment:
```suggestion
If you would prefer not to use the `sx` prop in your custom components, but only some parts of the system, you can use and combine the different style functions available.
```

Should we say why might a developer prefer this? (Better performance, smaller bundle size... ?)

Reply:
Done